client/components/splits/CardCapture.jsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-unused-vars */
+import React, { useEffect, useState } from 'react';
+
+import { Button } from '../../styles/GlobalStyle';
+
+const CardCapture = ({ advanceToConfirmSplit, advanceToRejectSplit }) => {
+  return (
+    <div>
+      <h2>Card Capture Step</h2>
```

Comment:
this would take in the setupIntent as well - only if they accept the split?

I wonder if card capture could just be it's own component we can just put in multiple places (like here and in between currentorder and currenttab)